ext/standard/tests/url/base64_decode_basic_002.phpt

```diff
@@ -21,21 +21,21 @@ var_dump(base64_decode($badChars));
 var_dump(base64_decode($badChars, false));
 var_dump(base64_decode($badChars, true));
 
-echo "Done";
+echo "Done\n";
 ?>
 --EXPECT--
-Decode 'hello world!':
-string(12) "hello world!"
-string(12) "hello world!"
-string(12) "hello world!"
+Decode 'PHP is a popular general-purpose scripting language that is especially suited to web development':
+string(96) "PHP is a popular general-purpose scripting language that is especially suited to web development"
+string(96) "PHP is a popular general-purpose scripting language that is especially suited to web development"
+string(96) "PHP is a popular general-purpose scripting language that is especially suited to web development"
```

Comment:
It's better not to change existing tests but create new once. Otherwise we have to check what is really changed.